pkg/cluster/bootstrap.go

```diff
@@ -309,17 +309,47 @@ func migrateBootstrapData(ctx context.Context, data io.Reader, files bootstrap.P
 
 const systemTimeSkew = int64(3)
 
+// isMigrated checks to see if the given bootstrap data
+// is in the latest format.
+func isMigrated(buf io.ReadSeeker) bool {
+	buf.Seek(0, 0)
+	defer buf.Seek(0, 0)
+
+	files := make(bootstrap.PathsDataformat)
+	if err := json.NewDecoder(buf).Decode(&files); err != nil {
```

Comment:
I feel a little weird about throwing away the result of a successful decode here - it feels like we could refactor things a bit so that the decoded results were returned and used if successful. That can wait though, in the interest of getting the issue fixed.